scipy/linalg/tests/test_lapack.py

```diff
@@ -1672,23 +1675,39 @@ def test_gttrf_gttrs():
 
         _dl, _d, _du, du2, ipiv, info = dgttrf(dl, d, du)
 
-        np.testing.assert_allclose(dl, diag_cpy[0], atol=atol)
-        np.testing.assert_allclose(d, diag_cpy[1], atol=atol)
-        np.testing.assert_allclose(du, diag_cpy[2], atol=atol)
-        
+        np.testing.assert_array_equal(dl, diag_cpy[0])
+        np.testing.assert_array_equal(d, diag_cpy[1])
+        np.testing.assert_array_equal(du, diag_cpy[2])
+
         b_cpy = b.copy()
         x_dgttrs, info = dgttrs(_dl, _d, _du, du2, ipiv, b)
-        np.testing.assert_allclose(x, x_dgttrs, atol=atol)
-        np.testing.assert_allclose(b, b_cpy, atol=atol)
-        
+        np.testing.assert_array_equal(b, b_cpy)
```

Comment:
I changed the wrapper on my machine so that b will be maintained, but I don't think it's implemented yet here.